stdlib/public/SDK/SceneKit/SceneKit.swift

```diff
@@ -151,7 +151,7 @@ extension SCNGeometryElement {
   /// Creates an instance from `indices` for a `primitiveType`
   /// that has a constant number of indices per primitive
   /// - Precondition: the `primitiveType` must be `.triangles`, `.triangleStrip`, `.line` or `.point`
-  public convenience init<IndexType : Integer>(
+  public convenience init<IndexType : FixedWidthInteger>(
```

Comment:
It does not require anything. Can as well be `BinaryInteger`, but I think the underlying implementation will be quite surprised if given something with a random memory representation, i.e. a 3rd party bignum.